### PR TITLE
Simplified Nextcloud logo icon

### PR DESCRIPTION
**Pull request by pixelipo**
This is part of the SVG optimization task #5157 

Nextcloud logo code has been simplified, but at the same time it's more human-readable.

I've changed the size to more traditional values (256*128). Circles are a bit fatter than they used to be (+0.5px at 128px width). If that is an issue, I can change their width to 21.5px.

`logo-icon.svg` is deprecated since new `logo.svg` is already much smaller in weight.

I've combined `.logo` and `.logo-icon` - this is not strictly needed, I can revert if @nextcloud/designers disagree. It requires adding `logo` class to tags that have `logo-icon` class in apps - mainly where pages are shared publicly, e.g. Calendar.

PNG images and favicons were not edited.

Signed-off-by: Marin Treselj <marin@pixelipo.com>

### CODE REVIEW

**Comment by MorrisJobke** [comment]
cc @nextcloud/designers

**Comment by MorrisJobke** [comment]
@pixelipo Could I ask you to resolve the conflict?

**Comment by jancborchardt** [comment]
We need to check this with theming also cc @nextcloud/theming

File: apps/theming/css/theming.scss [hunk 1]
```diff
@@ -42,12 +42,8 @@
 }
 
 /* override styles for login screen in guest.css */
-#header .logo,
-#header .logo-icon {
+#header .logo {
 	background-image: url(#{$image-logo});
-	@if $theming-logo-mime != '' {
```
Comment (juliushaertl) [inline apps/theming/css/theming.scss]: That should not be removed, otherwise we have https://github.com/nextcloud/server/issues/4585 again.

**Review by juliusknorr — CHANGES_REQUESTED** [review]

File: apps/theming/css/theming.scss (hunk 1 above)
Comment (jancborchardt) [inline apps/theming/css/theming.scss]: Yeah … when seeing the "contain" I was reminded of something too :D

**Comment by pixelipo** [comment]
@MorrisJobke I just managed to royally screw up this PR while attempting a rebase :-1:

I guess I'll have to create a new one...

**Comment by jancborchardt** [comment]
@pixelipo happens to all of us sometimes. ;) It's always best to test the rebase locally, then in case it's broken the last good version is still in the main repository. :)

File: apps/theming/css/theming.scss (hunk 1 above)
Comment (pixelipo) [inline apps/theming/css/theming.scss]: Well, if we want to limit this logo to 175px, as is defined [here](#diff-f6ba04997c6dcc9cec0389d9ef83e358L105), than the solution would be to simply change the width of the `.logo` element. Why is it set to 252px if it never fills that space?

**Comment by juliushaertl** [comment]
@pixelipo You should be able to just create a new branch from master and run `git cherry-pick de9088d` to pick the commit from this PR.

**Comment by pixelipo** [comment]
Replaced by #5407 @juliushaertl @jancborchardt please review over there

**Pull request closed without merging**